approval_frame/forms.py

```diff
@@ -17,3 +18,22 @@ class RegistrationFormCustom(RegistrationFormUniqueEmail):
         error_messages={'invalid': _("This value may contain only letters, "
             + "numbers and ./+/-/_ characters.")}
     )
+
+class PasswordResetFormCustom(PasswordResetForm):
```

Comment:
Actually, in the default PasswordResetForm, I could not find functionality where the form would error out in case the specific email address was not registered with the system. Instead, it would simply not send the email to that address.
So I added it to serve this purpose.  Le me know what you think.
Thanks!